### ISSUE TITLE
background data

### ISSUE THREAD

**Issue by mathematiciseverywhere** [sordonia/hred-qs#10]
Hello dear sordonia
you wrote;

 we sort the query log by query timestamp and we use the queries submitted before 1 May,
2006 as our background data to estimate the proposed model and the baselines. The next two weeks of data are used as a training set for tuning the ranking models. The remaining two weeks are split into the validation and the test set.

what does it mean background data? why didnt you use in training phase background data.
